### DISCUSSION TITLE
4.20.0之后menu更改，新增了items的类型ItemType[]这个类型中children的类型中可否增加一个自定义节点类型

### DISCUSSION THREAD

**Discussion by xiwadai** [ant-design/ant-design#36160] [Q&A]
4.20.0之后menu更改，新增了items的类型ItemType[]这个类型中children的类型中可否增加一个自定义节点类型，menu的配置文件可以单独抽出来一份和router路由公用一份，这样可以循环一下Router
![image](https://user-images.githubusercontent.com/24467843/174924333-c464a702-285d-45c8-b12a-fdcb9d194385.png)

**Comment by xhua007** [comment]
no one answer ? I have meet it,too
